src/main/java/org/opensearch/security/ssl/util/SSLConfigConstants.java

```diff
@@ -99,9 +99,9 @@ public final class SSLConfigConstants {
 
     public static final String JDK_TLS_REJECT_CLIENT_INITIATED_RENEGOTIATION = "jdk.tls.rejectClientInitiatedRenegotiation";
 
-    private static final String[] _SECURE_SSL_PROTOCOLS = { "TLSv1.3", "TLSv1.2", "TLSv1.1" };
+    private static final String[] _SECURE_SSL_PROTOCOLS = { "TLSv1.3", "TLSv1.2" };
 
-    public static final String[] getSecureSSLProtocols(Settings settings, boolean http) {
+    public static String[] getSecureSSLProtocols(Settings settings, boolean http) {
```

Comment:
for my understanding, why remove `final` modifier here and below?

Reply:
no point in `final static` for functions. In any case it is impossible to override them.